linpack_wrapper/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM registry.access.redhat.com/ubi8:latest
+
+RUN dnf install -y --nodocs git python3-pip python3-numpy python3-requests python3-numpy
```

Comment:
Minor nit but I don't think we need to install git w/ this unless I'm missing something.

Reply:
fixed.